pkg/virt-handler/hotplug-disk/mount.go

```diff
@@ -504,9 +508,10 @@ func (m *volumeMounter) updateBlockMajorMinor(dev uint64, allow bool, manager cg
 func (m *volumeMounter) createBlockDeviceFile(basePath *safepath.Path, deviceName string, dev uint64, blockDevicePermissions os.FileMode) error {
 	if _, err := safepath.JoinNoFollow(basePath, deviceName); errors.Is(err, os.ErrNotExist) {
 		return mknodCommand(basePath, deviceName, dev, blockDevicePermissions)
-	} else {
+	} else if err != nil {
 		return err
 	}
+	return nil
```

Comment:
nit: this can be kept as is. (If err is nil, it will return nil anyway).

```
} else {
     return err
}
```

Reply:
Done